### ISSUE TITLE
Update the quantity of a product in a suborder with more than one product

### ISSUE THREAD

**Issue by alex-jungwirth** [coreshop/MarketWarehouseBundle#4]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             | no

When trying to update the quantity of a product of a suborder with more than one product, the update for the suborder works, but the main order is affected. An exception is thrown during the request:

```
Timestamp: Fri Feb 02 2024 08:53:47 GMT+0100 (Mitteleuropäische Normalzeit)
Status: 500 | Internal Server Error
URL: /admin/coreshop/order/update/111540
Method: post
Message: Call to undefined method getQuantity in class Pimcore\Model\DataObject\CoreShopOrderPackage
Trace: 
in /var/www/html/vendor/pimcore/pimcore/lib/Model/AbstractModel.php:257
#0 /var/www/html/vendor/pimcore/pimcore/models/DataObject/AbstractObject.php(1212): Pimcore\Model\AbstractModel->__call()
#1 /var/www/html/vendor/coreshop/market-warehouse-bundle/src/Order/Processor/OrderPackageImmutableProcessor.php(53): Pimcore\Model\DataObject\AbstractObject->__call()
#2 [internal function]: CoreShop\Bundle\MarketWarehouseBundle\Order\Processor\OrderPackageImmutableProcessor::CoreShop\Bundle\MarketWarehouseBundle\Order\Processor\{closure}()
#3 /var/www/html/vendor/coreshop/market-warehouse-bundle/src/Order/Processor/OrderPackageImmutableProcessor.php(53): array_map()
#4 /var/www/html/vendor/coreshop/core-shop/src/CoreShop/Component/Order/Processor/CompositeCartProcessor.php(41): CoreShop\Bundle\MarketWarehouseBundle\Order\Processor\OrderPackageImmutableProcessor->process()
#5 /var/www/html/vendor/coreshop/core-shop/src/CoreShop/Bundle/OrderBundle/Manager/CartManager.php(95): CoreShop\Component\Order\Processor\CompositeCartProcessor->process()
#6 /var/www/html/vendor/coreshop/core-shop/src/CoreShop/Component/Pimcore/DataObject/VersionHelper.php(43): CoreShop\Bundle\OrderBundle\Manager\CartManager->CoreShop\Bundle\OrderBundle\Manager\{closure}()
#7 /var/www/html/vendor/coreshop/core-shop/src/CoreShop/Bundle/OrderBundle/Manager/CartManager.php(105): CoreShop\Component\Pimcore\DataObject\VersionHelper::useVersioning()
#8 /var/www/html/vendor/coreshop/market-warehouse-bundle/src/Order/CartManager.php(38): CoreShop\Bundle\OrderBundle\Manager\CartManager->persistCart()
#9 /var/www/html/src/Controller/CoreShop/Admin/OrderController.php(147): CoreShop\Bundle\MarketWarehouseBundle\Order\CartManager->persistCart()
#10 /var/www/html/vendor/coreshop/core-shop/src/CoreShop/Component/Pimcore/DataObject/InheritanceHelper.php(34): App\Controller\CoreShop\Admin\OrderController->App\Controller\CoreShop\Admin\{closure}()
#11 /var/www/html/src/Controller/CoreShop/Admin/OrderController.php(151): CoreShop\Component\Pimcore\DataObject\InheritanceHelper::useInheritedValues()
#12 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(163): App\Controller\CoreShop\Admin\OrderController->updateAction()
#13 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(75): Symfony\Component\HttpKernel\HttpKernel->handleRaw()
#14 /var/www/html/vendor/symfony/http-kernel/Kernel.php(202): Symfony\Component\HttpKernel\HttpKernel->handle()
#15 /var/www/html/public/index.php(36): Symfony\Component\HttpKernel\Kernel->handle()
#16 {main}
```

To reproduce this bug, create an order with at least 2 different products of the same supplier. Try to decrease the quantity inside the suborder of any product and save.

(see on our test env: orderId 101878, orderNumber LOY211-TEST)

**Comment by wolfsascha** [comment]
@dpfaffenbauer can you please have a look into it.

**Comment by dpfaffenbauer** [comment]
@alex-jungwirth @wolfsascha please try out this fix: https://github.com/coreshop/MarketWarehouseBundle/pull/5

**Comment by alex-jungwirth** [comment]
> @alex-jungwirth @wolfsascha please try out this fix: #5

Works!
